spec/graphql/static_validation/rules/argument_literals_are_compatible_spec.rb

```diff
@@ -65,4 +65,126 @@
     }
     assert_includes(errors, fragment_error)
   end
+
+  describe "null value" do
```

Comment:
one thing I'm missing is what happens if you have type `[Int]!` and you provide value `null`? Does it become `[null]`? or is it invalid? (I think `1` would become `[1]` in this above case)